components/DatasheetEditor.tsx

```diff
@@ -151,6 +151,7 @@ function CustomEditComponent({
         fullWidth
         onValueChange={handleNumberValueChange}
         value={typeof value === 'number' ? value : ''}
+        inputProps={{ 'aria-label': 'Input field' }}
```

Comment:
We can get the row label for these aria labels from `row.label` (`row` is already being passed to this CustomEditComponent). And `field` is either "value" or "notes" so this should be good for screenreaders:

```js
inputProps={{ 'aria-label': `${row.label} ${field}` }}
```